gdal/gdal_merge.xml

```diff
@@ -0,0 +1,211 @@
+<tool id="gdal_merge" name="GDAL Merge" version="@VERSION@">
+    <description>mosaics a set of images</description>
+    <macros>
+        <import>gdal_macros.xml</import>
+    </macros>   
+    <expand macro="gdal_requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        gdal_merge.py
+            -q        
+            #for $s in $series 
+                '${s.input}'
+            #end for
+            
+            -of '$of'
+            
+            #if $settings.advanced=='advanced'
+                
+                #if $settings.co!=''
+                    -co '$settings.co'
+                #end if
+                
+                #if $settings.ot!=''
+                    -ot '$settings.ot'
+                #end if
+                
+                #if $settings.condi_ps.pixelsize=='use_ps'
+                    -ps $settings.condi_ps.pixelsize_x $settings.condi_ps.pixelsize_y
+                #end if
+                
+                $settings.tap
+                
+                #if $settings.condi_ul_lr.ul_lr=='ul_lr'
+                    -ul_lr $settings.condi_ul_lr.ulx $settings.condi_ul_lr.uly $settings.condi_ul_lr.lrx $settings.condi_ul_lr.lry
+                #end if
+                
+                $settings.v
+                
+                $settings.separate
+                
+                $settings.pct
+                
+                #if $settings.n!='nodata_value'
+                    -n '$settings.n'
+                #end if
+                
+                #if $settings.a_nodata!='output_nodata_value'
+                    -a_nodata '$settings.a_nodata'
+                #end if
+                
+                #if $settings.init!=''
+                    -init '$settings.init'
+                #end if
+                
+            #end if
+            
+            -o '$output'
+        ]]></command>
+        
+    <inputs>
+        <repeat name="series" title="Input file">
```

Comment:
Any reason why this is not multiple="True" instead of the repeat?

Reply:
No, multiple works well